bats/tests/helpers/commands.bash

```diff
@@ -80,3 +80,6 @@ rdshell() {
 rdsudo() {
     rdshell sudo "$@"
 }
+wsl_exe() {
+    "$SYSTEM32/wsl.exe" "$@"
+}
```

Comment:
Let's call just `wsl.exe` and let the OS look it up in the `PATH`. We don't need a wrapper function for it.

We are using the full path for all utilities bundled with Rancher Desktop, but for OS utilities we assume they are on the `PATH`. Just like we use `cp` and don't write `/bin/cp`.